cellarium/ml/module/base_module.py

```diff
@@ -13,6 +14,8 @@ class BaseModule(torch.nn.Module, metaclass=ABCMeta):
     Base module for all cellarium-ml modules.
     """
 
+    __call__: Callable[..., torch.Tensor | None]
```

Comment:
This adds return type to the module call. In the `training_step` we expect it to be `Tensor` or `None`. Otherwise it returns `Any`.